src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,25 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <UserInfo user={post.user} />
```

Comment:
user can be undefined, let's render the component only when the user exists
```suggestion
        {post.user && <UserInfo user={post.user} />}
```